torch/testing/_internal/common_utils.py

```diff
@@ -102,9 +103,10 @@
 except ImportError:
     has_pytest = False
 
-
 NAVI_ARCH = ("gfx1030", "gfx1100", "gfx1101")
 
+HAS_HIPCC = torch.version.hip is not None and ROCM_HOME is not None
```

Comment:
Is this assumption good enough? Should we make it even more robust by actually checking for `hipcc` presence? Something like: `os.path.exists(f"{ROCM_HOME}/bin/hipcc")`

Reply:
yes it can be ..